### PR TITLE
Student grants announcement

### PR DESCRIPTION
**Pull request by petrasovaa**
image

### CODE REVIEW

**Review by wenzeslaus — APPROVED** [review]
Co-author's review

File: content/news/2023_11_09_Student_Grants_announced.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+### Why to apply?
+
+**Experience:** Gain hands-on experience in a thriving open-source community.<br>
+**Mentorship:** Work alongside experienced developers who will guide you throughout your journey.<br>
+**Contribution:** Your work will directly enhance the functionality and accessibility of GRASS GIS for users worldwide.<br>
+**Stipend:** Get up to 1000 USD.
+
+
+### How to apply?
+Express your interest in applying through the current GRASS community channels and identify a topic matching your interest. Look for ways to contribute to GRASS GIS that are related to the selected topic and submit a pull request to demonstrate your skills. Draft a proposal and share it to get feedback. For details and eligibility rules, check the [wiki](https://grasswiki.osgeo.org/wiki/Student_Grants).
```
Comment (veroandreo) [inline content/news/2023_11_09_Student_Grants_announced.md:28]: Where should candidates share their proposal?

Comment (petrasovaa) [reply content/news/2023_11_09_Student_Grants_announced.md:28]: That's on the wiki, so we can clarify it there if needed.

Comment (petrasovaa) [inline content/news/2023_11_09_Student_Grants_announced.md:28]:
```suggestion
Express your interest in applying through the current GRASS community channels and identify a topic matching your interest. Look for ways to contribute to GRASS GIS that are related to the selected topic and submit a pull request to demonstrate your skills. Draft a proposal and [share](https://grasswiki.osgeo.org/wiki/Student_Grants#How_to_apply) it to get feedback. For details and eligibility rules, check the [wiki](https://grasswiki.osgeo.org/wiki/Student_Grants).
```

**Review by veroandreo — APPROVED** [review]

**Pull request merged**